tests/integration/test_kinesis.py

```diff
@@ -46,44 +47,30 @@ def test_list_streams(self):
 
     @attr('slow')
     def test_can_put_stream_blob(self):
+        unique_data = str(uuid4())
         self.client.put_record(
-            StreamName=self.stream_name, PartitionKey='foo', Data='foobar')
+            StreamName=self.stream_name, PartitionKey='foo', Data=unique_data)
         # Give it a few seconds for the record to get into the stream.
         time.sleep(10)
```

Comment:
Do you think we need to sleep 10 seconds now before going into the wait loop? Or do you think we can just go straight into the waiter logic? Not really a big deal. Just read a little weird when I saw a `sleep()` followed by a `wait_for_stream_data()` as I would have assumed the `sleep()` would not be needed if there was already a wait logic.

Reply:
I'll go ahead and remove it.